packages/create-redwood-app/template/web/tsconfig.json

```diff
@@ -14,5 +14,5 @@
     "types": ["jest"],
     "jsx": "preserve",
   },
-  "include": ["src", "../.redwood/globals"]
-}
\ No newline at end of file
+  "include": ["src", "../.redwood/**/*"]
```

Comment:
Why did you have to change this?

Reply:
Because otherwise the declarations don't seem to be picked up (even though they're in typeRoots)

Reply:
So now that we have this here, do we need them in typeRoots too, or is that redundant now?

Reply:
I'm not sure tbh @Tobbe, I know that typeRoots has no impact on this particular declaration, but I didn't want to change it incase it actually works in other cases.

Reply:
IIRC @peterp previously had it as an include, or suggested I move it to include. But then we decided to have it in typeRoots because it's "more correct". But if we now have to have it here anyway, I think we should remove it from typeRoots. @peterp Do you remember any more details?

Reply:
My preference is also to remove typeRoots (if it doesn't break anything), because as per the docs [here](https://www.typescriptlang.org/tsconfig#typeRoots), it should automatically pick up types in `node_modules/@types`, but I don't know if it handles workspaces and hoisting correctly

Reply:
Since this is a "code quality" change, I'll create a different PR after I validate it @Tobbe if its ok with you? We don't need it to go into the patch release

Reply:
Looks like we need it :D https://github.com/microsoft/TypeScript/issues/33183

Reply:
A different PR is fine 👍 
That TS issue you linked is saying typeRoots aren't working with hoisted types, right?